lib/app_profiler/sampler/config.rb

```diff
@@ -5,28 +5,31 @@
 module AppProfiler
   module Sampler
     class Config
-      attr_reader :sample_rate, :paths, :cpu_interval, :backends_probability
+      attr_reader :sample_rate, :targets, :cpu_interval, :backends_probability
 
       SAMPLE_RATE = 0.001 # 0.1%
-      PATHS = ["/"]
+      TARGETS = ["/"]
       BACKEND_PROBABILITES = { stackprof: 1.0, vernier: 0.0 }
       @backends = {}
 
       def initialize(sample_rate: SAMPLE_RATE,
-        paths: PATHS,
+        targets: TARGETS,
         backends_probability: BACKEND_PROBABILITES,
         backends_config: {
           stackprof: StackprofConfig.new,
-        })
+        },
+        paths: nil)
 
         if sample_rate < 0.0 || sample_rate > 1.0
           raise ArgumentError, "sample_rate must be between 0 and 1"
         end
 
         raise ArgumentError, "mode probabilities must sum to 1" unless backends_probability.values.sum == 1.0
 
+        ActiveSupport::Deprecation.warn("passing paths is deprecated, use targets instead") if paths
```

Comment:
Hum, that method doesn't exist. https://api.rubyonrails.org/classes/ActiveSupport/Deprecation.html

Reply:
interesting, maybe it was was available in a previous version -- because tests do pass locally. Will change to `ActiveSupport::Deprecation.new`

Reply:
done https://github.com/Shopify/app_profiler/pull/134/commits/032840c25c3a16082eef85e977b417cec6ff26a9